Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/UserHelpers/UserIntegrationTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using kCura.Relativity.Client;
+using NUnit.Framework;
+using Relativity.API;
+using Relativity.Test.Helpers.SharedTestHelpers;
+
+namespace Relativity.Test.Helpers.NUnit.Integration.UserHelpers
+{
+	[TestFixture]
+	public class UserIntegrationTests
+	{
+		private IHelper testHelper;
+		private IServicesMgr servicesMgr;
+		private IRSAPIClient rsapiClient;
+
+		[OneTimeSetUp]
+		public void SetUp()
+		{
+			testHelper = new TestHelper(ConfigurationHelper.ADMIN_USERNAME, ConfigurationHelper.DEFAULT_PASSWORD);
+			servicesMgr = testHelper.GetServicesManager();
+			rsapiClient = servicesMgr.CreateProxy<IRSAPIClient>(ExecutionIdentity.System);
+		}
+
+		[OneTimeTearDown]
+		public void TearDown()
+		{
+			testHelper = null;
+			servicesMgr = null;
+			rsapiClient = null;
+		}
+
+		[Test]
+		public void CreateUserAndDeleteUserTest()
+		{
+			// Act
+			int _userArtifactId = Relativity.Test.Helpers.UserHelpers.CreateUser.CreateNewUser(rsapiClient);
```

Comment:
I don't know the specifics of the code but does it fail if the user already exists when creating or fail if the user doesn't exist when deleting?

As of late, Chandra has been giving out notes to always clean the environment first prior to the test, in this case, it would be to attempt to delete the user in the setup prior to the test running.

Reply:
@TommyBigNose Added a cleanup method

Reply:
Also, what are you checking here? Are you making sure that the user is created as in Assert.Userartif >0